packages/fxa-shared/subscriptions/stripe.ts

```diff
@@ -231,3 +259,12 @@ export const getSubscriptionUpdateEligibility: (
 
   return SubscriptionUpdateEligibility.INVALID;
 };
+
+export const getMinimumAmount = (currency: string): number => {
+  if (STRIPE_MINIMUM_CHARGE_AMOUNTS.hasOwnProperty(currency))
```

Comment:
Any reason you prefer `hasOwnProperty` over `STRIPE_MINIMUM_CHARGE_AMOUNTS[currency]`?

Reply:
I think ESLint has feelings on this as well; https://eslint.org/docs/rules/no-prototype-builtins
But interestingly, <kbd>eslint --print-config</kbd> reports that the "no-prototype-builtins" rule is set to `"error"` currently.

```sh
npx eslint --config=_dev/.eslintrc.json --print-config packages/fxa-shared/subscriptions/stripe.ts | jq --sort-keys '.rules["no-prototype-builtins"]' | npx prettier --parser=json
# ["error"]
```

**NOTE:** But I have a feeling that we're not running ESLint against .ts/.tsx files, so not sure this would have been caught by husky/lint-staged.
I need to write up a Google doc on my limited understanding of the current state of linting in this repo.

Reply:
I was playing around trying to get JS to throw an error when it can't find a key in an object, and I think `hasOwnProperty` just stuck around from after that. (I was trying to avoid the `if` all together. Any ideas how to do that?)

I'll change it to your suggestion though. Does make it easier to read.